simple_token_authentication.gemspec

```diff
@@ -1,28 +1,52 @@
-$:.push File.expand_path("../lib", __FILE__)
+# -*- encoding: utf-8 -*-
+# stub: simple_token_authentication 1.15.0 ruby lib
 
-# Maintain your gem's version:
-require "simple_token_authentication/version"
-
-# Describe your gem and declare its dependencies:
 Gem::Specification.new do |s|
-  s.name        = "simple_token_authentication"
-  s.version     = SimpleTokenAuthentication::VERSION
-  s.authors     = ["Gonzalo Bulnes Guilpain"]
-  s.email       = ["gon.bulnes@gmail.com"]
-  s.homepage    = "https://github.com/gonzalo-bulnes/simple_token_authentication"
-  s.summary     = "Simple (but safe) token authentication for Rails apps or API with Devise."
-  s.license     = "GPL-3.0+"
+  s.name = "simple_token_authentication"
+  s.version = "1.15.0"
```

Comment:
Not related to the core issue, but just for you to know, you can require dependencies locally when you debug:

```ruby
# Gemfile

# ...

gem 'simple_token_authentication', path: '../simple_token_authentication' # assuming you cloned it next to your project
```

It's pretty convenient since you don't need to `bundle update` anymore when you make changes. It can come handy : )